### PR TITLE
Changed to show real memory usage

### PR DESCRIPTION
**Pull request by dimon222**
Basically solution for #57

### CODE REVIEW

**Comment by dimon222** [comment]
Interesting, it does look significantly different from my Debian output:
image

Perhaps, we should use platform dependent solution? Is there example for other platforms by any chance? I pretty much expect same behavior from Ubuntu (like I have), but not sure about the rest distributions.

**Comment by dimon222** [comment]
Try this new hybrid version

**Comment by willemdh** [comment]
Seems good solution to make it os dependant. Just need to check this on a few servers. Got a few opensuse servers too. Might take a week.

**Comment by willemdh** [comment]
@dimon222 Ok, I found some time to further test this. It still wasn't ok for CentOS / Red Hat. Changed it to this now where I make the sum of buff/cache and used:

```
    if [[ "$(jq -r ".Release.Value" $ExportFile)" =~ (^CentOS*|^Red*) ]] ; then
        # CentOS based distributions
        MemBuffCacheB="$(free | awk '/Mem:/ { print $6 }')"
        MemFreeUsedB="$(free | awk '/Mem:/ { print $3 }')"
        MemUsedB=$(echo "$MemBuffCacheB+$MemFreeUsedB" | bc)
        MemFreeB="$(free | awk '/Mem:/ { print $4 }')"
    else
        # Debian based distributions and possibly others (?)
        MemFreeB="$(free | grep buffers/cache | awk '{print $4}')"
        MemUsedB="$(free | grep buffers/cache | awk '{print $3}')"
    fi
```

**Comment by dimon222** [comment]
No worries, I'll commit this change shortly (in this PR) and squash everything together to avoid too much commits after merge.

**Comment by dimon222** [comment]
Actually, after small investigation I found out that its not system dependent but rather procps version related.
The two-line format is fresh new notation that seem to come beginning version 3.3.10.

Source: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=798994

I will update my changes with check for of /usr/bin/free version instead of basing it off OS distributive.

**Comment by willemdh** [comment]
Nice find @dimon222 , and nice VersionComparison function. I'll double check asap.

File: FireMotD [hunk 1]
```diff
@@ -334,9 +366,20 @@ ExploreMemory () {
     ExportTime=$(date '+%Y-%m-%d %H:%M:%S,%3N')
     MemTotalB="$(< /proc/meminfo grep MemTotal | awk '{print $2}')"
     MemTotalGB="$(printf "%0.2f\n" "$(bc -q <<< scale=2\;"$MemTotalB"/1024/1024)")"
-    MemFreeB="$(< /proc/meminfo grep MemFree | awk '{print $2}')"
+    ProcpsVersion="$(free --version | awk '{ print $4 }')"
+    IsItNew="$(VersionComparison "$ProcpsVersion 3.3.9")"
+
+    if [[ "$IsItNew" -lt 2 ]] ; then
```
Comment (dimon222) [inline FireMotD]: Seems like always True, something is wrong over there

**Comment by dimon222** [comment]
Found condition bug, fixed + rebased again. Should be working correctly for you now.

**Pull request merged**